### ISSUE TITLE
Super Mario RPG - 0100BC0018138000

### ISSUE THREAD

**Issue by RupeeClock** [Ryujinx/Ryujinx-Games-List#4811]
### Game Name

Super Mario RPG

### Game Update Version

1.0.0

### Emulator Build Version

1.1.1089

### Graphics Backend

OpenGL and Vulkan

### Hardware Specs

- ##### CPU: *12th Gen Intel(R) Core(TM) i7-12700K 3.60 GHz*
- ##### GPU: *NVIDIA RTX 3070)*
- ##### RAM: *32GB DDR5 (2x16gb)*

### Game State

3D remake of the 1996 SNES role-playing game by Nintendo and SquareSoft, built in Unity engine.
Game plays on-par with original Switch hardware, including minor performance hiccups (noticeable during vertical camera panning at Bowser's Keep). No performance issues encountered. No sound or pre-rendered cutscene playback issues.
Visual issues with some characters and environment geometry appearing improperly illuminated, looking dull and flat.

Disabling emulated console VSync will not improve performance, it can however introduce controller latency, delaying player inputs. This can make it difficult to time action commands for attacking and blocking during battle.

### Screenshots

![Screenshot (99)](https://github.com/Ryujinx/Ryujinx-Games-List/assets/24753760/ae42dc8f-c8bf-4998-be9a-ec14a8567d41)
![Screenshot (91)](https://github.com/Ryujinx/Ryujinx-Games-List/assets/24753760/2fb73633-6883-45ab-8f25-4ad1be44ed0c)
![Screenshot (92)](https://github.com/Ryujinx/Ryujinx-Games-List/assets/24753760/214bbfdb-a60b-42fa-943f-eb509233ebe8)
![Screenshot (96)](https://github.com/Ryujinx/Ryujinx-Games-List/assets/24753760/463f99e1-b2fd-45e5-8633-79a6dde0d45e)

### Log File

1.1.1089 Vulkan: [Ryujinx_1.1.1089_2023-11-17_09-14-53.log](https://github.com/Ryujinx/Ryujinx-Games-List/files/13390204/Ryujinx_1.1.1089_2023-11-17_09-14-53.log)
1.1.1089 OpenGL: [Ryujinx_1.1.1089_2023-11-17_09-34-48.log](https://github.com/Ryujinx/Ryujinx-Games-List/files/13390207/Ryujinx_1.1.1089_2023-11-17_09-34-48.log)

**Comment by Jomanakk** [comment]
do you have this issue where the first castle cutscene has some stuttering?

**Comment by Dejizoku** [comment]
For me all Cutscenes run slow but not the audio which causes it to be way out of sync. Does somebody know how to fix this?

**Comment by strider81** [comment]
I am having problems with the game crashing when I
![Screenshot (143)](https://github.com/Ryujinx/Ryujinx-Games-List/assets/151166653/4578aae3-1e73-4c56-a13c-62c50201d4d4)
 am following geno in the woods.. I am using vulkan does anyone have a solution?

**Comment by MetrosexualGarbodor** [comment]
Please post a log when talking about a crash, a screenshot does nothing for anyone

**Comment by strider81** [comment]
> Please post a log when talking about a crash, a screenshot does nothing for anyone

yes your majesty sorry i am new to this

**Comment by RupeeClock** [comment]
Regarding the castle cutscene with stuttering.
I'm referring to when Mario first approaches Bowser's Keep when Exor (the giant sword character) has pierced it, there is noticeable frame stutter as the camera pans up to show Exor. This occurs on both hardware and on the emulated system.
I've produced recordings using a Genki ShadowCast with OBS for Switch hardware, along with a GeForce Now recording of Ryujinx v1.1.1089. Note that the ShadowCast has slightly bright and oversaturated color output.
https://github.com/Ryujinx/Ryujinx-Games-List/assets/24753760/56d06822-a162-416e-8dea-d130fe53c858
https://github.com/Ryujinx/Ryujinx-Games-List/assets/24753760/bde1bb9b-079b-4ffc-bd94-a5f605ff3b37
This save file can be used to quickly access this scene: [SMRPG Save - Mario's Pad.zip](https://github.com/Ryujinx/Ryujinx-Games-List/files/13393983/SMRPG.Save.-.Mario.s.Pad.zip)

Concerning pre-rendered video cutscenes.
My machine does not experience any desync in these cutscenes and appears to match real hardware.
I've also produced recordings for these, with the Ryujinx recording showing my current settings.
https://github.com/Ryujinx/Ryujinx-Games-List/assets/24753760/45f25a9a-8eea-4a33-96b4-81b48c5c6946
https://github.com/Ryujinx/Ryujinx-Games-List/assets/24753760/ed13afb3-b5b4-412c-8c2a-4beef3c57b2d
This save file can be used to quickly trigger the pre-rendered cutscene introducing Mallow: [SMRPG Save - Pre-Mallow Pre-Rendered Cutscene.zip](https://github.com/Ryujinx/Ryujinx-Games-List/files/13394014/SMRPG.Save.-.Pre-Mallow.Pre-Rendered.Cutscene.zip)

Hopefully the settings in this video clip may help you troubleshoot performance/desync issues with the pre-rendered cutscenes?
The software appears to run audio separately to the video file, such as music and sound effects.

**Comment by AguileraJosue** [comment]
While playing Super Mario RPG I had the same problem as the comment above. In the Forest Maze, where you have to follow Geno. It crashes.
In my case I managed to get to Geno and the game was saved, but now when I try to load the game it crashes. I attach the log.
[Ryujinx_1.1.1089_2023-11-17_12-59-37.log](https://github.com/Ryujinx/Ryujinx-Games-List/files/13394177/Ryujinx_1.1.1089_2023-11-17_12-59-37.log)

**Comment by Bonesalone** [comment]
My game crashes when trying to enter down the pipe after beating Smilex in bean valley, any idea on what could be causing it or what to do?
Here's a screenshot of the console when the crash occurs
![Screenshot 2023-11-16 210934](https://github.com/Ryujinx/Ryujinx-Games-List/assets/151228354/97341dde-30c8-42b0-8692-a16efc3a1747)

**Comment by murphmaster01** [comment]
Game crashes during the cutscene in mushroom kingdom after you return peach to the chancellor. Tried different backends and other settings. 
[Ryujinx_1.1.1089_2023-11-17_19-43-03.log](https://github.com/Ryujinx/Ryujinx-Games-List/files/13398533/Ryujinx_1.1.1089_2023-11-17_19-43-03.log)

**Comment by MetrosexualGarbodor** [comment]
@murphmaster01 your game file is broken, dump it again and ask on Discord for help next time

**Comment by MetrosexualGarbodor** [comment]
@AguileraJosue your game file is broken, dump it again and ask on Discord for help next time

**Comment by EmaGeist** [comment]
### Bug Report

### What's the issue you encountered?
The pre-rendered cutscenes run slow, audio is normal. As a result they are completely out of sync. This happens on both OpenGL and Vulkan.
Speaking of cutscenes, the game crashes after the cutscene where Mario gets sent flying out of Bowser's Castle at the start of the game. It crashes on Vulkan but it doesn't on OpenGL.

### How can the issue be reproduced?
For the first issue, just play any pre-rendered cutscene. Apparently this doesn't happen to everyone.
As for the second issue, try both backends. On Vulkan this is fixed by enabling Texture Recompression in the settings.

### Environment?
Ryujinx version: Ryujinx 1.1.1089
Game version: 1.0.0
System Specs:
OS: Win 11
CPU: AMD Ryzen 7 4800H with Radeon Graphics
GPU: AMD Radeon(TM) RX 5600M Series
RAM: 16GB
Applied Mods : No
[DxDiag.txt](https://github.com/Ryujinx/Ryujinx-Games-List/files/13398529/DxDiag.txt)

### Log Files:
1.1.1089 Vulkan: 
[Ryujinx_1.1.1089_2023-11-17_18-34-10.log](https://github.com/Ryujinx/Ryujinx-Games-List/files/13398544/Ryujinx_1.1.1089_2023-11-17_18-34-10.log)
1.1.1089 OpenGL: 
[Ryujinx_1.1.1089_2023-11-17_18-35-43.log](https://github.com/Ryujinx/Ryujinx-Games-List/files/13398546/Ryujinx_1.1.1089_2023-11-17_18-35-43.log)

**Comment by Testsr** [comment]
For cards with slow video decoding, expect a desync with sound. This is because your card is attempting to decode the video at the same time as playing the audio, which is much easier.

**Comment by Dejizoku** [comment]
It is not just a problem for slow cards, I have a RTX 3060 Ti and the same slow video is happening for me

**Comment by RupeeClock** [comment]
I believe my RTX 3070 and your RTX 3060 Ti should have the same video decoding capabilities.
I was able to reproduce the cutscene slowdown by emulating SMRPG whilst my system was running a VRAM intensive task (generating images using stable-diffusion-webui).
Under these conditions, performance warnings were frequently captured in logs: 
```HLE.OsThread.10 ServiceNv Wait: GPU processing thread is too slow, waiting on CPU...```
Whenever these occurred during video playback, video would stutter and fall behind audio playback, or may playback slowly.
This appears to be likely to occur when the GPU's dedicated memory is fully utilised, falling back to main RAM.

I observed that when SMRPG loads a pre-rendered video cutscene, that VRAM usage can increase by around 1GB.
This might hit a memory usage threshold on some hardware, though my graphics card's VRAM doesn't come close to capacity under normal Ryujinx usage.

As SMRPG features pre-rendered video running at 60 FPS at what may be a significant bitrate, this will be more demanding to decode in-time than other Switch titles that typically have 30 FPS pre-rendered video.

These ideas come to mind for troubleshooting these video cutscene playback issues, external to Ryujinx settings:
* Assess the amount of VRAM your GPU has
* Check if other resource intensive applications are running, and close them
    * Use the Performance section of Task Manager to monitor your GPU's memory usage
* Ensure both graphics card and integrated graphics drivers, along with other hardware drivers are up to date
* Run Ryujinx and dumped game data from the fastest storage devices in your system
   * In other words, avoid running from mechanical HDDs or USB drives
* Ensure the integrity of your dumped game files
* If using a 12th Generation or later Intel Core series processor on **Windows 10**, disable efficient cores in BIOS settings
   * Windows 10 is not able to differentiate efficient cores from performance cores, and causes a huge performance impact

I am also wondering if there are some unanticipated variables as to why some users report slow video cutscenes, such as if they are running multiple monitors. I only have a single-monitor setup.

**Comment by Kidboy17** [comment]
I'm not experiencing any issues with the game.
CPU: 11th Gen Intel i5-11600k
GPU: Nvidia 4070 ti
Ram: 16gb (2x8gb)
OS: Win 10

I also have ReBar Size turned on in my BIOS. I've noticed that improved a lot of the PC games I play and it might also be contributing to having no issues with Super Mario RPG.

**Comment by samuelbirocchi** [comment]
The game crashes when I enter a pipe in Bean Valley.

**Comment by RupeeClock** [comment]
> The game crashes when I enter the last pipe in Bean Valley.
> How can I debug this?

Check Ryujinx log files after a crash occurs, and share the log with any crash reports.
If your log file contains something to the effect of the following, it means your game dump file is faulty and needs redumping:
`HLE.OsThread.8 Application : Unhandled exception caught: System.Reflection.TargetInvocationException: Exception has been thrown by the target of an invocation.
 ---> LibHac.Common.HorizonResultException: ResultFsNonRealDataVerificationFailed (2002-4604): Hash error!`

**Comment by PeaceIsSoftcoreWar** [comment]
The game runs well enough but the audio is consistently distorted.

Spec List:
CPU: Ryzen 7700X
GPU: RX 7900 XTX
RAM: 32GB of DDR5 Ram
Ryujinx Version 1.1.1092
OS: Windows 10
Graphics Driver: 23.11.1

Setting List:
Shader Cache and Macro HLE; Enabled.
Texture Recompression: Off
Graphics backend: Vulkan and OpenGL
Resolution Scale: 2x
Post Processing Effect: None
Upscale: Nearest
Anisotropic Filtering: Auto
Aspect Ratio: 16:9

**Comment by GamerzHell9137** [comment]
> The game runs well enough but the audio is consistently distorted.
> 
> Spec List: CPU: Ryzen 7700X GPU: RX 7900 XTX RAM: 32GB of DDR5 Ram Ryujinx Version 1.1.1092 OS: Windows 10 Graphics Driver: 23.11.1
> 
> Setting List: Shader Cache and Macro HLE; Enabled. Texture Recompression: Off Graphics backend: Vulkan and OpenGL Resolution Scale: 2x Post Processing Effect: None Upscale: Nearest Anisotropic Filtering: Auto Aspect Ratio: 16:9

Change Windows sound settings to 2 Channel 16bit 48 Khz.

**Comment by Mario9699** [comment]
Game Update Version : 1.0.0
Current on master : Emulator Build Version : 1.1.1092
Graphics Backend : OpenGL
OS: Windows 10
CPU: i5-8400
GPU: NVIDIA GeForce GTX 1050
RAM: 16GB

This game sound have distortion
[Ryujinx_1.1.1092_2023-11-22_20-09-31.log](https://github.com/Ryujinx/Ryujinx-Games-List/files/13439217/Ryujinx_1.1.1092_2023-11-22_20-09-31.log)

**Comment by stenovera** [comment]
I'm going to throw my hat in regarding slow cutscenes and desynched audio, as I also have these issues. Game runs perfectly at 60fps otherwise, no desynced audio. I've been checking it out ever since 1.1.1089, and have not moved past the Mallow cutscene to help reference it. My settings were already the same as in RupeeClock's post, so unsure what it could be.
One of the other posters mentioned potential issues with a second monitor, and I do usually run a second monitor. As it happens, I was gonna test it on 1.1.1093, but the game has disappeared from my list and cannot be booted as of the time of this posting.
Game Update Version: 1.0.0
Current on master: Emulator Build Version : 1.1.1092
Graphics Backend: Vulkan
OS: Windows 10
CPU: AMD Ryzen 9 5900HS 
GPU: NVIDIA GeForce RTX 3080
RAM: 16GB

**Comment by RupeeClock** [comment]
@stenovera 
It looks like you're using a capable high-end gaming laptop, judging by the CPU.
The pre-rendered cutscene slowdown/desync occurs when your systems's CPU is struggling to emulate the Switch's nvdec (video decoding) capabilities. To mitigate this, you'll want to maximise CPU performance, as well as ensure GPU performance is high. 

You may want to check your Windows configuration so that you can maximise performance, such as:
* Checking or customising your Power Plan
* Changing your Power Mode to Best Performance on the battery icon in task bar
* Changing Graphic Settings and specifying a Graphices Preference of your NVIDIA GPU for the Ryujinx application

I also recommend using NVIDIA Control Panel to manage the 3D settings for the Ryujinx application, I change these settings:
* Low Latency Mode: On
     * SMRPG has strict timing on inputs, this can reduce input/render latency
* OpenGL rendering GPU: NVIDIA
* Power Management Mode: Prefer Maximum Performance
* Vertical Sync: On
    * I find this helps to prevent frame stutter, even when Ryujinx emulation speed is 100%; it also serves as a frame cap based on your display's refresh rate (applicable for some emulated Switch software when Switch VSync is disabled)
* Triple buffering: On

Finally, for the matter of the game disappearing from your list, check if the game files are still where you left them on your system. You can still boot a game from its file without using the games list.

**Comment by Mario9699** [comment]
Game Update Version : 1.0.0
Current on master : Emulator Build Version : 1.1.1093
Graphics Backend : OpenGL
OS: Windows 10
CPU: i5-8400
GPU: NVIDIA GeForce GTX 1050
RAM: 16GB

This game have flask back
[Ryujinx_1.1.1093_2023-11-23_23-15-46.log](https://github.com/Ryujinx/Ryujinx-Games-List/files/13451524/Ryujinx_1.1.1093_2023-11-23_23-15-46.log)

**Comment by stenovera** [comment]
I gave the changed settings a shot, and the game still has slow cutscenes and desynched sounds, sadly. I was monitoring the GPU during gameplay and it hit 100% throughout. Did not have other GPU applications on at the time. I did confirm that disconnecting my other display improved performance, but not by much. It feels like a emulator issue, or maybe my hardware isn't strong enough/isn't configured well enough. Still do not have the other issues listed by people here.

**Comment by samuelbirocchi** [comment]
> > The game crashes when I enter the last pipe in Bean Valley.
> > How can I debug this?
> 
> Check Ryujinx log files after a crash occurs, and share the log with any crash reports. If your log file contains something to the effect of the following, it means your game dump file is faulty and needs redumping: `HLE.OsThread.8 Application : Unhandled exception caught: System.Reflection.TargetInvocationException: Exception has been thrown by the target of an invocation. ---> LibHac.Common.HorizonResultException: ResultFsNonRealDataVerificationFailed (2002-4604): Hash error!`

Yeah the problem was with the dump, dumped again and it worked.

I was testing with the latest Ryujinx version and it appears to have improved the lag in the cutscenes.

**Comment by Viperxoc** [comment]
Game Update Version : 1.0.0
Current on master : Emulator Build Version : 1.1.1093
Graphics Backend : Vulkan
OS: Windows 11
CPU: AMD Ryzen 7 6800HS
GPU: AMD Radeon RX 6800S
RAM: 16GB

Crash during the after the castle scene.

[Ryujinx_1.1.1093_2023-11-24_10-47-42.log](https://github.com/Ryujinx/Ryujinx-Games-List/files/13461156/Ryujinx_1.1.1093_2023-11-24_10-47-42.log)
[Ryujinx_1.1.1093_2023-11-24_10-45-46.log](https://github.com/Ryujinx/Ryujinx-Games-List/files/13461157/Ryujinx_1.1.1093_2023-11-24_10-45-46.log)

**Comment by OldManKain** [comment]
@Viperxoc @Mario9699 your game file is broken, dump it again and ask on Discord for help next time.

**Comment by dmjohn0x** [comment]
> I gave the changed settings a shot, and the game still has slow cutscenes and desynched sounds, sadly. I was monitoring the GPU during gameplay and it hit 100% throughout. Did not have other GPU applications on at the time. I did confirm that disconnecting my other display improved performance, but not by much. It feels like a emulator issue, or maybe my hardware isn't strong enough/isn't configured well enough. Still do not have the other issues listed by people here.

I was having this Audio Issue on Star Ocean R. The way I solved it was simply changing my audio backend to SDL2 in the Setting>System menu. I hope this helps for you.

**Comment by whatslinucks** [comment]
I'm running into the same bug when running Super Mario RPG on Ryujinx that is seen here in a Yuzu bug report: https://github.com/yuzu-emu/yuzu/issues/12155
Game Update Version : 1.0.1 (issue exhibits also in 1.0.0)
Ryujinx Build Version : 1.1.1104
Graphics Backend : Vulkan (exhibits same issue in OpenGL as well)
OS: Windows 10
CPU: i7-9700K
GPU: NVIDIA GeForce RTX 3070
RAM: 16GB

Final messages in the Ryujinx when the soft lock occurs are:
00:05:37.281 |W| HLE.OsThread.10 ServiceNv Wait: GPU processing thread is too slow, waiting on CPU...
00:05:38.281 |E| HLE.OsThread.10 Gpu WaitOnSyncpoint: Wait on syncpoint 1 for threshold 65708 took more than 1000ms, resuming execution...
00:06:05.219 |I| .NET TP Worker Ptc Save: Saved Profiling Info (size: 156236 bytes, profiled functions: 45118).

**Comment by Shihta** [comment]
I had freeze issue on macOS
I tried 1.1.1102 and 1.1.1102 + [this PR](https://github.com/Ryujinx/Ryujinx/pull/6068), both of them have freeze issue.

log:
[Ryujinx_1.1.1+0_2024-01-12_20-14-49.log](https://github.com/Ryujinx/Ryujinx-Games-List/files/13991268/Ryujinx_1.1.1%2B0_2024-01-12_20-14-49.log)
[Ryujinx_1.1.1+0_2024-01-12_20-05-02.log](https://github.com/Ryujinx/Ryujinx-Games-List/files/13991270/Ryujinx_1.1.1%2B0_2024-01-12_20-05-02.log)

then I tried [new JIT](https://github.com/Ryujinx/Ryujinx/pull/6057), but it still has freeze issue

log:
[Ryujinx_1.1.1+0_2024-01-19_18-58-53.log](https://github.com/Ryujinx/Ryujinx-Games-List/files/13991275/Ryujinx_1.1.1%2B0_2024-01-19_18-58-53.log)

my environment:

MacBook Air M2 2022
RAM: 24G
macOS version: 14.1.1（23B81）

**Comment by KamranNosrati** [comment]
i had no issue till the forest maze,when i try to progress any further the emulator crashes
should i dump the game? and how exactly can i do that?

**Comment by KamranNosrati** [comment]
For anyone having crashing issues, disabling the “Enable FS Integrity Checks” option in setting>system, will hopefully fix the corruption error.

**Comment by Shihta** [comment]
https://github.com/Ryujinx/Ryujinx/pull/6356 seems to fix the freeze issue I mentioned earlier
I think I played for more than 5 hours and didn't have this issue again

**Comment by 4rmanDC** [comment]
This game is no longer running.

**Comment by Zoomer1988** [comment]
I'd like to chime in since I'm too experiencing slow video cutscenes.

The system is Asus Zephyrus G15 2022 model with AMD Ryzen mobile 9 5900 HS with RTX 3080.

Never ever faced any slowdowns in Ryujinx, but, as OP mentions it, it's the first with 60fps video, I believe. So the symptoms are the same. Audio is fine, and the video plays slow (as in slowmo, no jerking) and lags behind the further it goes.

I use external 60hz monitor (TV actually) with DP video out so as to avoid the built-in graphics and muxers altogether (this system does allow that and if the internal display is disabled, the integrated AMD graphics doesn't do anything, unlike previous Zephyrus notebooks where it isn't possible to avoid display muxing).

What I have tried so far without success:

Switching between Vulkan and OpenGL backends (no change)
Toggling VSync (the video gets a little bit more jerky, but overal behaviour is the same)
Changing memory manager mode (no change)
Switching to different audio backends (SDL2, SoundIO - audio is fine, OpenAL - audio stutters as if buffer underrun but still on time, video is still behind)
Toggling Enable Macro HLE (no change)
Toggling Graphical backend multithreading (no change)
Messing around with different graphical settings in Nvidia Control Planel for Ryujinx profle - threading, vsync and other stuff - and no change)
I also made sure that all possible power saving features are disabled and maximum performance plans are being used. The notebook is connected to AC power. Dedicated GPU is chosen whenever possible. No GPU intensive apps are running in background.

The system's overall performance is also kept in check - with thermals and other stuff. Performs just like similar laptop configs, i.e., as expected. In-game the fps is stable, locked and no slowdowns. In other games too.

So I'm at a loss here. I'm attaching a screenshot with Windows' task manager - it doesn't seem like the GPU or CPU is struggling - or does it?

Also, during the cutscene I'm getting a constant spam in the console log with the following line (like 30 times a second or more - you can see that by timestamps):

00:00:42.485 |S| HLE.OsThread.19 ServiceMm Get: Stubbed. {requestId: 2}
00:00:42.553 |S| HLE.OsThread.19 ServiceMm SetAndWait: Stubbed. {requestId: 2 ; clockRateMin: 429488704 ; clockRateMax: -1}
00:00:42.553 |S| HLE.OsThread.19 ServiceMm Get: Stubbed. {requestId: 2}
00:00:42.591 |S| HLE.OsThread.19 ServiceMm SetAndWait: Stubbed. {requestId: 2 ; clockRateMin: 429488704 ; clockRateMax: -1}
00:00:42.591 |S| HLE.OsThread.19 ServiceMm Get: Stubbed. {requestId: 2}
00:00:42.622 |S| HLE.OsThread.19 ServiceMm SetAndWait: Stubbed. {requestId: 2 ; clockRateMin: 429488704 ; clockRateMax: -1}

It stops the second the video stops decoding.

I also get one "00:02:11.809 |W| HLE.OsThread.4 ServiceNv Wait: GPU processing thread is too slow, waiting on CPU..." but that's just once, and it's during in-game scene load long after the cutscene ends.

I'm trying to run the game in native res, no special mods or anything except a localization patch, though I tried without it and it's still the same.

So I don't think that the system performance is to blame. However, it still looks like it's the system config problem, since not everyone is experiencing this issue. I'm not sure what's wrong, though. I've tried eliminating all those edge-case things with high-refresh monitors and the like. I'd be really grateful for any assistance since this is the game I'd like to run with patched-in localization - so no go for the console.

[Ryujinx_1.1.1334_2024-06-19_16-00-32.log](https://github.com/user-attachments/files/15902477/Ryujinx_1.1.1334_2024-06-19_16-00-32.log)

![Screenshot 2024-06-19 152654](https://github.com/Ryujinx/Ryujinx-Games-List/assets/6277416/ee4b692f-c3c6-4791-9d60-2aae26b145bc)
![Screenshot 2024-06-19 155913](https://github.com/Ryujinx/Ryujinx-Games-List/assets/6277416/40cebb92-ebcc-489f-8527-92ee1266115b)

**Comment by Zoomer1988** [comment]
Yeah. It seems so. I also decided to test in the latest available version of another emulator and here's a couple of observations:

There's a hardware acceleration of nvenc streams available there. However, even with hardware decoding enabled there's still pretty noticeable audio desync. Although, I'm not sure it's related, since video does run full speed with it enabled. Possibly desync happens somewhere else.

However! There's an option to enable software decoding and with it the video still seems to be running full speed. There's still desync, but it's not that noticeable and I still feel that it happens due to some other emulation issue.

So here's that.

**Comment by RupeeClock** [comment]
The audio desync would be because the video cutscenes don't have any audio in the actual video files; all audio comes from the game engine instead, so it relies on both the video playback and game engine emulation emulating accurately without performance hitches.
Since emulation isn't to a precise standard like the original hardware, especially with everyone's PCs having widely varying components, it's hard to get this behaviour correct outside of throwing as much processing power as you can at the application.
